lab-trent/lib/constants.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
```

Comment:
Check out the NPM package called dotEnv. I think that what you're trying to do here would be simplified with that package.